include/aws/cal/symmetric_cipher.h

```diff
@@ -35,6 +35,8 @@ typedef struct aws_symmetric_cipher *(aws_aes_gcm_256_new_fn)(
 typedef struct aws_symmetric_cipher *(
     aws_aes_keywrap_256_new_fn)(struct aws_allocator *allocator, const struct aws_byte_cursor *key);
 
+enum aws_symmetric_cipher_state { AWS_CIPHER_READY, AWS_CIPHER_FINALIZED, AWS_CIPHER_ERROR };
```

Comment:
nit: probably spell out AWS_SYMMETRIC_CIPHER_* to avoid any collisions in the future, since we likely would want to do that for other cases